package.json

```diff
@@ -14,12 +14,12 @@
     "packages/*"
   ],
   "scripts": {
-    "prestart": "yarn bootstrap",
-    "start": "yarn workspace docs-app start",
-    "start:watch": "yarn workspace docs-app start:watch ",
-    "dev": "yarn start:watch",
+    "prestart": "npm run bootstrap",
```

Comment:
This is behaving differently from before. when I run `npm start dev` it runs this one too, so it will take ages.

```
matyas.szabo@c02dg48pmd6t instructure-ui % npm start dev

> instructure-ui@8.43.0 prestart
> npm run bootstrap


> instructure-ui@8.43.0 bootstrap
> node scripts/bootstrap.js

cleaning packages...
lerna notice cli v7.1.3
clean finished
lerna notice filter including "@instructure/ui-icons"
lerna info filter [ '@instructure/ui-icons' ]

> @instructure/ui-icons:prepare-build
...
```

Same for `npm run start:examples`

Reply:
for the first one I think you should do `npm run dev` not `npm start dev`

Reply:
for the second, it runs bootstrap because we have this line in package.json: `"prestart:examples": "npm run bootstrap"`

Reply:
so this seems to be intended

Reply:
All the scripts including `npm run dev` is pretty fast for me. Arguably faster than before